src/frontend/app/features/services/services-wall/services-wall.component.ts

```diff
@@ -46,6 +49,11 @@ export class ServicesWallComponent implements OnDestroy {
       this.cfOrgSpaceService,
       serviceInstancesSchemaKey,
       'all').subscribe();
+
+    this.haveConnectedCf$ = cloudFoundryService.cFEndpoints$.pipe(
+      map(endpoints => endpoints.map(endpoint => endpoint.connectionStatus === 'connected')),
+      map(connected => connected.reduce((a, v) => a || v, false))
```

Comment:
Something like `!!connected.find(isConnected => isConnected)` would be more efficient.

Reply:
Or even just `!!endpoints.find(endpoint => endpoint.connectionStatus === 'connected'))`

Reply:
Yeah - I just used what we had in the CF page - but can have a look at simplifying